shared/js/ui/views/whitelist.es6.js

```diff
@@ -99,7 +95,9 @@ Whitelist.prototype = window.$.extend({},
         [this.$remove, 'click', this._removeItem],
         [this.$add, 'click', this._addItem],
         [this.$showadd, 'click', this._showAddToWhitelistInput],
-        [this.$url, 'keyup', this._manageInputChange]
+        [this.$url, 'keyup', this._manageInputChange],
+        // listen to changes to the whitelist model
+        [this.store.subscribe, 'change:whitelist', this.rerender]
```

Comment:
I read better the comments in the [store](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/e3b2c9be1c6a852b705a0dcbe482fb9de2dd552e/shared/js/ui/base/store.es6.js) file, and when listening to changes, we listen to the whole model, not just to a property (unlike what we're used to) - so in this case it only works when listening to change:whitelist because whitelist is the name of the model!